ui/components/molecules/M007-AdressFormFragment/index.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+import createPropTypes from 'json-schema-prop-types'
+
+import Grid from '@material-ui/core/Grid'
+
+import TextFieldLabel from '../../atoms/A010-TextFieldLabel'
+
+import schema from './schema'
+
+const M007 = props => {
+  return (
+    <Grid container spacing={8}>
+      <Grid item xs={12}>
+        <TextFieldLabel
```

Comment:
we should Probably have addressName as well in case the address name is different to the persons actual name eg c/o enspiral dev academy